store/sqlstore/channel_store.go

```diff
@@ -1814,6 +1815,70 @@ func (s SqlChannelStore) UpdateLastViewedAt(channelIds []string, userId string)
 	return times, nil
 }
 
+// CountUnreadSince gives the number of posts that a channel has since some date, this doesn't follow any threads.
```

Comment:
I'm not sure what this means when it says it doesn't follow threads. Also, we might want to clarify that this uses the `CreateAt` timestamp since there's some other "Since" methods that are based off the `UpdateAt` timestamp since we want to see anything that changed
```suggestion
// CountUnreadSince gives the number of posts in a channel created since the given time.
```